src/palindrom.js

```diff
@@ -759,7 +767,7 @@
   };
 
   Palindrom.prototype.showWarning = function (heading, description) {
-    if (this.debug && global.console && console.warn) {
+    if (this.debug && console && console.warn) {
```

Comment:
The above line will cause an error if run in an environment where console is undefined. Try this in Chrome Dev Tools:

<img width="384" alt="screen shot 2017-03-21 at 14 04 54" src="https://cloud.githubusercontent.com/assets/566463/24148499/621879d0-0e3f-11e7-851c-191c0ad00dec.png">